macos/generate_wazuh_packages.sh

```diff
@@ -57,6 +57,9 @@ function build_package() {
     # create package
     if packagesbuild ${AGENT_PKG_FILE} --build-folder ${DESTINATION} ; then
         echo "The wazuh agent package for MacOS X has been successfully built."
+        if [[ "${CHECKSUM}" = "yes" ]]; then
+            shasum  -a512 ${DESTINATION}/${pkg_name} > ${DESTINATION}/${pkg_name}.sha512
```

Comment:
There's an extra whitespace in `shasum -a512`

Reply:
True, thanks: https://github.com/wazuh/wazuh-packages/pull/192/commits/aecd138582fb44d9537afac8671577c306fa73b6